sw/airborne/subsystems/intermcu/intermcu_fbw.c

```diff
@@ -66,10 +66,21 @@ static void intermcu_parse_msg(void (*commands_frame_handler)(void));
 static void checkPx4RebootCommand(unsigned char b);
 #endif
 
+#ifdef USE_GPS
+#include "subsystems/abi.h"
+#include "subsystems/gps.h"
+static abi_event gps_ev;
+static void gps_cb(uint8_t sender_id, uint32_t stamp, struct GpsState *gps_s);
+#endif
+
 void intermcu_init(void)
 {
   pprz_transport_init(&intermcu.transport);
 
+#if USE_GPS
+  AbiBindMsgGPS(ABI_BROADCAST, &gps_ev, gps_cb);
```

Comment:
And using ABI_BROADCAST here, gps_cb gets actually called twice per new GPS message: once from the actual GPS implementation and once from gps.c with GPS_MULTI_ID